src/scripts/seqbox_utils.py

```diff
@@ -323,14 +354,16 @@ def read_in_sample_source_info(sample_source_info):
 def read_in_culture(culture_info):
     check_cultures(culture_info)
     culture = Culture()
-    culture.date_cultured = culture_info['date_cultured']
+    culture.date_cultured = datetime.datetime.strptime(culture_info['date_cultured'],'%d/%m/%Y')
     culture.culture_identifier = culture_info['culture_identifier']
     culture.submitter_plate_id = culture_info['submitter_plate_id']
     culture.submitter_plate_well = culture_info['submitter_plate_well']
     return culture
 
 
 def read_in_extraction(extraction_info):
+    # extraction_info['date_extracted'] = extraction_info['date_extracted'].to_pydatetime()
```

Comment:
delete things like this before merging, unless they're really useful. if they're useful, add a comment explaining why.